astropy/cosmology/core.py

```diff
@@ -777,6 +791,46 @@ def nu_relative_density(self, z):
 
         return prefac * self._neff_per_nu * rel_mass
 
+    def _get_nufunc_scalar(self):
+        """ Return the version of the nu relative density function
+        to use in integrals."""
+
+        if self._massivenu:
+            return self._massivenu_relative_density_scalar
+        else:
+            return lambda x : 0.22710731766 * self._Neff
```

Comment:
Okay--maybe if you need speedups at that level though it would be worth investigating a Cython rewrite :)
If nothing else a comment would be nice.